manticore/manticore.py

```diff
@@ -394,7 +394,7 @@ def wrapper(*args, **kwargs):
 
     @property
     def running(self):
-        return self._executor._running.value
```

Comment:
`_executor.running` is a SyncManager.Value(). don't we want to return the `.value`, not the Value() directory?

Reply:
whoops ignore